src/components/PostComment.jsx

```diff
@@ -0,0 +1,40 @@
+import {useState, useContext} from "react"
+import "./postComment.css"
+import {UserContext} from "../contexts/User"
+import { Login } from "../contexts/Login"
+import { postComment } from "../api"
+function PostComment({comments, setComments, article_id}){
+    const [comment, setComment] = useState("")
+    const {user, setUser} = useContext(UserContext)
+    const {setLogin} = useContext(Login)
+    const loginPop = document.getElementsByClassName("login-form")
```

Comment:
You shouldn't be interacting with the DOM directly! Can you conditionally render the pop up depending on a condition in state?